dcc-portal-ui/package.json

```diff
@@ -21,7 +21,10 @@
     "test": "karma start karma.conf.js",
     "extract-text": "angular-gettext-cli --files 'app/scripts/**/*.+(html|js)' --dest translations/strings.pot",
     "compile-text": "angular-gettext-cli --compile --files 'translations/*.po' --dest 'app/scripts/translations/js/translations.js'",
-    "lint": "eslint -c config/eslint.js --no-eslintrc app/vendor app"
+    "lint": "eslint -c config/eslint.js --no-eslintrc app/vendor app",
+    "ensure-versions": "npm run ensure-node-version && npm run ensure-npm-version || echo 'Please ensure your version of node and npm satisfies' `json -f package.json engines` && exit 10",
+    "ensure-node-version": "semver `node -v` -r `json -f package.json engines.node`",
```

Comment:
a library to parse validity of version numbers

counting on it to throw an error if the version number from `npm -v` and `node -v` do not satisfy what's in the package.json